2023/11/06 04:55:19 failed to dial tcp connection, retrying
2023/11/06 04:55:20 failed to dial tcp connection, retrying
[2023-11-06T04:55:20.881Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-11-06T04:55:20.883Z] info  Using user-data-dir ~/.local/share/code-server
[2023-11-06T04:55:20.895Z] info  Using config file ~/.config/code-server/config.yaml
[2023-11-06T04:55:20.896Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-11-06T04:55:20.896Z] info    - Authentication is disabled
[2023-11-06T04:55:20.896Z] info    - Not serving HTTPS
====================================>token: 621f0d3a-a2ab-4824-bd62-f8d005933d48
[04:55:24] 




[04:55:24] Extension host agent started.
[04:55:25] [127.0.0.1][8983a1df][ManagementConnection] New connection established.
[04:55:26] [127.0.0.1][007cabe1][ExtensionHostConnection] New connection established.
[04:55:26] [127.0.0.1][007cabe1][ExtensionHostConnection] <378> Launched Extension Host Process.
[04:55:29] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[04:55:46] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-11-06T04:57:36.697Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
[05:03:59] [127.0.0.1][8983a1df][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[05:03:59] [127.0.0.1][007cabe1][ExtensionHostConnection] <378> Extension Host Process exited with code: 0, signal: null.
2023/11/06 05:14:00 exit program after timeout
